addons/main/functions/anomalies/fn_activateTeleport.sqf

```diff
@@ -30,11 +30,8 @@ if (count _teleporters < 2) exitWith {
 };
 
 private _exit = objNull;
-{
-    if (_trg != _x) then {
-        _exit = _x;
-    };
-} forEach _teleporters;
+_otherTeleporters = _teleporters - [_trg];
+_exit = selectRandom _otherTeleporters;
```

Comment:
```suggestion
_exit = selectRandom (_teleporters - [_trg]);
```

Why, no need to assign a one time use variable if the content of the variable is short enough.